SpiffWorkflow/bpmn/serializer/workflow.py

```diff
@@ -123,6 +125,8 @@ def serialize_json(self, workflow, use_gzip=False):
     def deserialize_json(self, serialization, read_only=False, use_gzip=False):
         dct = json.loads(gzip.decompress(serialization)) if use_gzip else json.loads(serialization)
         version = dct.pop('serializer_version')
+        if version in MIGRATIONS:
```

Comment:
I guess in the future this might get more complicated, as you would run 1 to 1.1 then 1.1 to 1.2 ....  But seems overkill to fix that at the moment.

Reply:
I thought about this a fair amount, but ended up going with the simplest solution, because a complicated solution that attempts to anticipate issues might not anticipate the right ones and then end up merely being complicated without solving any actual problems.

I'm not sure 1.0 -> 1.1, 1.1 -> 1.2, etc is that bad (though we should probably turn the migrations file into it's own package if we accumulate a lot of changes).  We can automate it pretty easily for people who are not customizing the serializer, and it's better to have discrete pieces of code that anyone who does can use or extend.